lib/web/apiserver.go

```diff
@@ -2111,6 +2114,34 @@ func (h *Handler) clusterNodesGet(w http.ResponseWriter, r *http.Request, p http
 	}, nil
 }
 
+type getLoginAlertsResponse struct {
+	Alerts []types.ClusterAlert `json:"alerts"`
+}
+
+// clusterLoginAlertsGet returns a list of on-login alerts for the user.
+func (h *Handler) clusterLoginAlertsGet(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+	// Get a client to the Auth Server with the logged in user's identity. The
+	// identity of the logged in user is used to fetch the list of alerts.
+	clt, err := ctx.GetUserClient(site)
```

Comment:
Should this client be closed when we're done with it?

Reply:
It wasn't in any of the other endpoints but if you feel it's necessary I can add it.

Reply:
@Joerger what do you think? I'm not super familiar with these clients, but my guess is that it's probably an oversight and if there's a method that releases resources we should probably use it.

Reply:
Auth clients are cached in the session and they're closed when the session gets cleaned up, AFAIK.